cli/src/main.rs

```diff
@@ -49,7 +51,7 @@ async fn main() -> Result<(), color_eyre::Report> {
     }
 
     if let Ok(genesis_path) = std::env::var("IROHA2_GENESIS_PATH") {
-        args.genesis_path = std::path::PathBuf::from_str(&genesis_path)?;
+        args.genesis_path = Some(std::path::PathBuf::from_str(&genesis_path)?);
     }
```

Comment:
If we use [the script](https://github.com/ilchu/iroha/blob/173a0e3f2c9796f776d1ebec20a9a4209df05f18/scripts/test_env.sh#L83) or [docker](https://github.com/ilchu/iroha/blob/173a0e3f2c9796f776d1ebec20a9a4209df05f18/Dockerfile#L59) with `IROHA2_GENESIS_PATH` set, Iroha enters [this line](https://github.com/ilchu/iroha/blob/173a0e3f2c9796f776d1ebec20a9a4209df05f18/core/src/genesis.rs#L271) and panics.

Possible workarounds would be:
- Set the environment variable only for the genesis peer
- Open the json file only for the genesis peer

Reply:
Fixed by putting it into the `if is_submit..` clause.